Assets/Scripts/GameManager/GameManager.cs

```diff
@@ -0,0 +1,18 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class GameManager : MonoBehaviour
+{
+    // Start is called before the first frame update
```

Comment:
Same comment for this one as last

Reply:
also removed, part of the tutorial that isn't necessary  for this request.